nodes/matrix/matrix_out_mk2.py

```diff
@@ -80,7 +85,7 @@ def update_mode(self, context):
         items=mode_items, default="AXISANGLE", update=update_mode)
 
     def sv_init(self, context):
-        self.inputs.new('SvMatrixSocket', "Matrix").is_mandatory =True
+        self.sv_new_input('SvMatrixSocket', "Matrix", dict(is_mandatory=True, nesting_level=2))
```

Comment:
```suggestion
        self.sv_new_input('SvMatrixSocket', "Matrix", is_mandatory=True, nesting_level=2)
```
I see this as more convenient API